installer/lib/phoenix_new.ex

```diff
@@ -184,7 +188,7 @@ defmodule Mix.Tasks.Phoenix.New do
 
       # Configure your database
       config :#{binding[:application_name]}, #{binding[:application_module]}.Repo,
-        adapter: Ecto.Adapters.Postgres,
+        adapter: #{binding[:adapter_module]},
```

Comment:
You want to `inspect binding[:adapter_module]` otherwise it will be `Elixir.Ecto.Adapters.Postgres`.